admin_pages/transactions/templates/txn_admin_details_side_meta_box_billing_info.template.php

```diff
@@ -22,9 +22,9 @@ function ee_show_billing_info_cleaned(EE_Form_Section_Proper $form_section, $fou
             ?>
             <div class="clearfix">
             <span class="admin-side-mbox-label-spn lt-grey-txt float-left">
-                    <?php echo esc_html($subsection->get_html_for_label()); ?>
+                    <?php echo $subsection->get_html_for_label(); ?>
```

Comment:
sigh... well that was dumb of me to escape the output of a function that is clearly returning HTML 😞